modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/java/org/nuxeo/apidoc/browse/ApiBrowserConstants.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.apidoc.browse;
+
+/**
+ * @since 11.1
+ */
+public class ApiBrowserConstants {
```

Comment:
Could be `ApiBrowsers` as there's a method + final + private constructor.

Reply:
wdym? changing the class name?

Reply:
this is pure cleanup and i don't find the new name better --> kept it as is, please tell if that's an issue